Docs/Configuration.tex

```diff
@@ -3068,7 +3090,7 @@ \subsection{Boot Properties}\label{miscbootprops}
   \item \texttt{0x0001} --- \texttt{OC\_ATTR\_USE\_VOLUME\_ICON}, provides custom icons
     for boot entries:
 
-    For \texttt{Tools} OpenCore will try to load a custom icon and fallback to the default icon:
+    For \texttt{Tools}, OpenCore will attempt loading custom icons and fallback to a default icon on failure:
```

Comment:
Is it still clear that each tool can have one icon at most?

Reply:
Amended to singular